torch/autograd/__init__.py

```diff
@@ -152,5 +152,5 @@ def grad(outputs, inputs, grad_outputs=None, retain_graph=None, create_graph=Non
         outputs, grad_outputs, retain_graph,
         inputs, only_inputs)
 
-status = torch._C._autograd_init()
-assert status, "Autograd failed to initialize."
+if not torch._C._autograd_init():
```

Comment:
I thought this was ok this way as there are many places in the code where if an assert is not respected, the rest of the code will fail. Isn't the point of the optimized version to actually disable all these check assuming they will not fail.